src/extension/obs.ts

```diff
@@ -22,6 +22,7 @@ const videosCharity = nodecg.Replicant<Asset[]>('assets:videos-charity');
 const videosSponsors = nodecg.Replicant<Asset[]>('assets:videos-sponsors');
 const videosLong = nodecg.Replicant<Asset[]>('assets:videos-long');
 const hosterkaReplicant = nodecg.Replicant<Hosterka>('hosterka');
+const hostMuteStatusReplicant = nodecg.Replicant<Boolean>('hostMuteStatus');
```

Comment:
```suggestion
const hostMuteStatusReplicant = nodecg.Replicant<boolean>('hostMuteStatus');
```

Use `boolean` instead of `Boolean`
Reason why: 
> The type names String, Number, and Boolean (starting with capital letters) are legal, but refer to some special built-in types that will very rarely appear in your code. Always use string, number, or boolean for types.

https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#the-primitives-string-number-and-boolean

Reply:
TIL